walkthroughs/week-5-tdd/intro/src/main/java/com/google/sps/Greeter.java

```diff
@@ -22,6 +22,11 @@
    * Returns a greeting for the given name.
    */
   public String greet(String name) {
+    name = removeWhitespaces(name);
     return "Hello " + name;
   }
+
+  public String removeWhitespaces(String string) {
```

Comment:
If the string is "Sean Gilpin" this should produce "SeanGilpin" correct? Just want to confirm thats what you want.

Reply:
That's correct.